backends/ebpf/tests/p4testdata/action-profile2.p4info.txtpb

```diff
@@ -92,15 +92,5 @@ actions {
     bitwidth: 16
   }
 }
-action_profiles {
-  preamble {
-    id: 298015716
-    name: "MyIC.ap"
-    alias: "ap"
-  }
-  table_ids: 39967501
-  table_ids: 47318070
-  size: 1024
-}
```

Comment:
Do you know why this change in this file is here?

Is this a file that is being diff'ed against actual compiler output files during automated CI tests, it seems like either (a) this change is a result of code changes in this PR, which I am very confused about, or (b) this should have been causing CI failures before.

Reply:
That change is a mistake, I will correct it once I am back on a dev machine. The CI is catching this failure correctly. 
The main problem is that the `ptf-ebpf` appear not have an automated way to replace test files, so there is a decent amount of tedium involved in updating them.